docs/api.md

```diff
@@ -0,0 +1,523 @@
+# Loki's HTTP API
+
+Loki exposes an HTTP API for pushing, querying, and tailing log data.
+Note that [authenticating](operations/authentication.md) against the API is
+out of scope for Loki.
+
```

Comment:
This is where I would document which component of Loki is responsible for which endpoint. I think it is important for people who want configure Loki in microservices mode and want to have some proxy in front of all those services.

Reply:
Great idea, done.